web/themes/custom/bevan/bevan.theme

```diff
@@ -121,9 +121,11 @@ function bevan_preprocess_status_messages(&$variables) {
   if (!empty($variables['message_list']['error']) && count($variables['message_list']['error']) > 0) {
     /** @var \Drupal\Core\Render\Markup $lastMessage */
     $lastMessage = end($variables['message_list']['error']);
-    $haystack = $lastMessage->__toString();
-    if ((strpos($haystack, '1 error has been found') > -1) && (strpos($haystack, 'Email') > 0)) {
-      array_pop($variables['message_list']['error']);
+    if (!isset($lastMessage)) {
```

Comment:
Good spot. I wonder, since the `end()` [returns the last element or FALSE for an empty array](https://www.php.net/manual/en/function.end.php#refsect1-function.end-returnvalues) I wonder if this `if()` can just be `if ($lastMessage) { … }`.

I'm not quite sure of the conditions needed to get the error this patch fixes, but could be useful to check it can be that simple. (It's just that I always look at a `!isset()` or a `!empty()` and get a bit unsettled by the double negative, but sometimes it's the only way, and maybe that's needed here.)

Reply:
Fixed with code in follow-on commits